srcalc/src/test/java/gov/va/med/srcalc/domain/model/SampleModels.java

```diff
@@ -146,25 +164,56 @@ public static Specialty thoracicSpecialty()
         	    new Specialty(62, "Vascular")
                 );
     }
+    
+    /**
+     * Constructs a VariableGroup object with the given properties.
+     * @param name the group name
+     * @param displayOrder the group display order
+     * @return a new VariableGroup object with a mock database ID of
+     * displayOrder + 1
+     */
+    private static VariableGroup makeVariableGroup(
+            final String name, final int displayOrder)
+    {
+        final VariableGroup vg = new VariableGroup(name, displayOrder);
+        // Normally only Hibernate calls the package-private setId() but fake it
+        // here.
+        vg.setId(displayOrder + 1);
+        return vg;
+    }
 
     public static VariableGroup procedureVariableGroup()
     {
-        return new VariableGroup("Planned Procedure", 0);
+        return makeVariableGroup("Planned Procedure", 0);
     }
     
     public static VariableGroup demographicsVariableGroup()
     {
-        return new VariableGroup("Demographics", 1);
+        return makeVariableGroup("Demographics", 1);
     }
     
     public static VariableGroup labVariableGroup()
     {
-        return new VariableGroup("Laboratory Values", 4);
+        return makeVariableGroup("Laboratory Values", 4);
```

Comment:
I kept the display orders in line with the actual data, but I just left out some groups in the sample set. Do you think this is an issue?